pandas/io/tests/test_sql.py

```diff
@@ -1346,9 +1369,35 @@ def test_double_precision(self):
         self.assertTrue(isinstance(col_dict['i32'].type, sqltypes.Integer))
         self.assertTrue(isinstance(col_dict['i64'].type, sqltypes.BigInteger))
 
+    def test_connectable_issue_example(self):
+        """
+        This tests the example raised in issue
+        https://github.com/pydata/pandas/issues/10104
+        """
```

Comment:
Can you add these comments as normal comments? (using `#`) 
Problem is that otherwise the docstring masks the test name in the output of the testsuite